FWCore/ParameterSet/src/ConfigurationDescriptions.cc

```diff
@@ -176,18 +177,19 @@ namespace edm {
         << "plugin name = \"" << pluginName << "\"  label name = \"" << labelAndDesc.first << "\"\n";
     }
 
-    std::string cfi_filename;
-    if (0 == strcmp(baseType.c_str(),kSource)) {
-      cfi_filename = pluginName + "_cfi.py";
```

Comment:
I took the liberty of simplifying a bit the logic here (and below) since for services the `labelAndDesc.first == pluginName` is guaranteed already by the check above.

Reply:
Argh, it was `kSource`, not `kService`.